docs/commands.md

```diff
@@ -0,0 +1,101 @@
+# Commands
+> List of available commands
+
+- #### **[`init`](#init--i)**
+- #### **[`generate`](#generate--g)**
+  - [`--type`](#--type--t)
+  - [`--mutation`](#--mutation--m)
+  - [`--loader`](#--loader--l)
+  - [`--connection`](#--connection--c)
+- #### **[`help`](#--help--h)**
+- #### **[`version`](#--version--V)**
+
+## `init`, `-i`
+Provides an easy way to create a GraphQL server based on [@entria/graphql-dataloader-boilerplate](https://github.com/entria/graphql-dataloader-boilerplate) which is a production-ready server
+
+> We are currently using the same boilerplate on three applications running on production at **[@entria](https://github.com/entria)**
+
+```sh
+create-graphql init GraphQLProject
+```
+Creates a new GraphQL project. The project contains the following structure:
+```
+├── /data/                   # GraphQL generated schema
+├── /repl/                   # Read-Eval-Print-Loop (REPL) configuration
+├── /scripts/                # Generate GraphQL schema script
+├── /src/                    # Source code of GraphQL Server
+│   ├── /connection/         # Connections types (Relay)
+│   ├── /interface/          # NodeInterface (Relay)
+│   ├── /loader/             # Loaders of the models using DataLoader
+│   ├── /model/              # Models definition (Mongoose, SQL, Google DataStore)
+│   ├── /mutation/           # Mutations definition
+├── /test/                   # Test helpers
+```
+
+## `generate`, `-g`
+Generates the files by passing these options:
+
+- [`--type`](#--type--t)
+- [`--mutation`](#--mutation--m)
+- [`--loader`](#--loader--l)
+- [`--connection`](#--connection--c)
+
+#### `--type`, `-t`
```

Comment:
I think that we need to show only the information. If the user need the details, he can try to execute the commands to see them.